Add package readmes

Hornwitser opened the pull request:
Adds README.md files for all the published packages and a description of the available configuration settings.

psihius commented:
I was thinking that these probably should be linked to from the main readme file, otherwise you kind'a have to know to search for these.

Hornwitser commented:
Hmm, well yes.  But this sort of applies to all of the files in docs/.  The project readme needs a clean up too.

Hornwitser commented:
I added links in the main readme file, updated the description of Clusterio in it to be a lot less out of date, added a section of configuring the master server port, setting it up behind a proxy, and removed claims of supporting Node.js v11 as that's EOL.
